### PR TITLE
Fixing res.send to save only stringified objects to cache.

### PR DESCRIPTION
**Pull request by RSNara**
Changes: 
- Express's `response.send` returns the response stream itself. In our monkey-patch, we should do the same. 
- If the `body` passed to `response.send` is an object and not a buffer, `response.send` calls `response.json`, which later calls `response.send` to send the data to the client. This means that when we monkey-patch `response.send`, we end up saving to the cache twice. The first time, we save an object, the second, we save a string. Seeing as though all objects are converted to string eventually, it looks to me like we should only have the need to save strings in our redis cache.

### CODE REVIEW

File: lib/ExpressRedisCache/route.js [hunk 1]
```diff
@@ -179,10 +183,13 @@ module.exports = (function () {
                 /** Create the new cache **/
                 self.add(name, body, {
                     type: this._headers['content-type'],
-                    expire: expirationPolicy(res.statusCode)
```
Comment (co2-git) [inline lib/ExpressRedisCache/route.js:182]: just out of curiosity, why did you remove that part?

**Comment by co2-git** [comment]
Thanks for submitting :)

File: lib/ExpressRedisCache/route.js (hunk 1 above)
Comment (RSNara) [inline lib/ExpressRedisCache/route.js:182]: Oops, sorry. That part shouldn't have been removed. I'll add it in again.

**Comment by co2-git** [comment]
:+1:

**Comment by RSNara** [comment]
Is there a certain procedure we need to go through to have this merged?

**Comment by co2-git** [comment]
hey @rv-kip i'll merge @ramanpreetnara's PR

**Comment by co2-git** [comment]
@ramanpreetnara i'll update the npm package as well. Cheers mate :+1:

**Comment by rv-kip** [comment]
Thanks @co2!

**Comment by RSNara** [comment]
thanks!
